androguard/core/bytecode.py

```diff
@@ -885,6 +885,7 @@ def FormatClassToPython(i):
     :rtype: str
     """
     i = i[:-1]
+    print(i)
```

Comment:
is the print here necessary?  Better use logging.debug()

Reply:
No, I forgot to remove it